File: MPI/mpicheck/tests/functions/1.0/MPI_Pack_size.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
        call mpi_pack_size(var_0, var_1, var_2, var_3, var_4)
       call pmpi_pack_size(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Pack_size.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Datatype var_1;
    MPI_Comm var_2;
    int *var_3;
    int ret;
    /* calls */
    ret = MPI_Pack_size(var_0, var_1, var_2, var_3);
    ret = PMPI_Pack_size(var_0, var_1, var_2, var_3);
    return 0;
}
